gstreaming:
  ros__parameters:
    video_dev: "/dev/video4"
    # video_file: "/home/lxsky/ucl-cs/y4/fyp/3d-scanner/images/vid19.mp4"
    fps_rescale: 20
    default_on: false
    single_step: false

orb_slam3:
  ros__parameters:
    orb_voc: "data/orb_slam/ORBvoc.txt"
    cam_cfg: "data/orb_slam/webcam_hikvision_dse12.yaml"
    fps_low: 5

depth_extract:
  ros__parameters:
    weights: "data/simplerecon/hero_model.ckpt"

/**:
  ros__parameters:
    input_img_w: 1920
    input_img_h: 1080
    depth_img_w: 1280
    depth_img_h: 720
    cx: 1007.89
    cy: 547.17
    fx: 1346.31
    fy: 1011.07
    res_recon: 0.01
    res_process: 0.05
